### PR TITLE
(docs) Release note (v2.10)

### PR DESCRIPTION
**Pull request by Louis-J**
#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: docs/source/release.rst [hunk 1]
```diff
@@ -5,6 +5,48 @@
...
+""""""""""
+
+*  Fixed an error in HPO experiment hyper-parameters page.
+*  Fixed a link error in About menu on WebUI.
+
+Known issues
+""""""""""""
+
+*  Modelspeedup does not support non-tensor intermediate variables
+*  The training service runs much slower after a while.
```
Comment (liuzhe-lz) [inline docs/source/release.rst:48]: Don't include this line. There's nothing to do with NNI.

File: docs/source/release.rst [hunk 2]
```diff
@@ -5,6 +5,48 @@
...
 ==========
 
+Release 2.10 - xx/xx/xxxx
+-------------------------
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*  Added deduplication for evolutionary search.
+*  Fixed the racing issue in RL strategy submit model.
```
Comment (QuanluZhang) [inline docs/source/release.rst:15]: submit model -> on submitting models?

File: docs/source/release.rst [hunk 3]
```diff
@@ -5,6 +5,48 @@
...
+*  Fixed an issue introduced by the trial recovery feature.
+*  Fixed import error of ``Lightning`` in nas.
```
Comment (QuanluZhang) [inline docs/source/release.rst:17]: ``Lightning`` should be ``PyTorch Lightning``. nas should be NAS

File: docs/source/release.rst [hunk 4]
```diff
@@ -5,6 +5,48 @@
...
+
+Compression
+"""""""""""
+
+*  #3514
```
Comment (QuanluZhang) [inline docs/source/release.rst:22]: a little strange to simply put an id here

File: docs/source/release.rst [hunk 5]
```diff
@@ -5,6 +5,48 @@
...
+*  Fixed the bug that speedup ``rand_like_with_shape`` is easy to overflow when ``dtype=torch.int8``.
+*  Fixed the bug when calling ``torch._C.parse_schema`` in pytorch < 1.9.0.
+*  Fixed the propagation error with view tensors in speedup.
```
Comment (QuanluZhang) [inline docs/source/release.rst:25]: -> propagation error of ``torch.view`` operator ?

File: docs/source/release.rst [hunk 6]
```diff
@@ -5,6 +5,48 @@
...
+
+Hyper-parameter optimization
+""""""""""""""""""""""""""""
+
+*  Fixed a dependency issue of Anneal tuner by change Anneal tuner dependency to optional.
```
Comment (QuanluZhang) [inline docs/source/release.rst:30]: change -> changing

File: docs/source/release.rst [hunk 7]
```diff
@@ -5,6 +5,48 @@
...
+*  Fixed a bug that tuner might loss connection in long experiments.
```
Comment (QuanluZhang) [inline docs/source/release.rst:31]: loss -> lose

File: docs/source/release.rst [hunk 8]
```diff
@@ -5,6 +5,48 @@
...
+
+Training service
+""""""""""""""""
+
+*  Fixed a bug that trial code directory cannot have non-English characters.
+
+Web portal
+""""""""""
+
+*  Fixed an error in HPO experiment hyper-parameters page.
```
Comment (QuanluZhang) [inline docs/source/release.rst:41]: what is this error? better to very briefly describe it

File: docs/source/release.rst [hunk 9]
```diff
@@ -5,6 +5,48 @@
...
+"""""""""""
+
+*  #3514
+*  Fixed the bug that speedup ``rand_like_with_shape`` is easy to overflow when ``dtype=torch.int8``.
+*  Fixed the bug when calling ``torch._C.parse_schema`` in pytorch < 1.9.0.
+*  Fixed the propagation error with view tensors in speedup.
+
+Hyper-parameter optimization
+""""""""""""""""""""""""""""
+
```
Comment (QuanluZhang) [inline docs/source/release.rst:29]: could add one more: Supported rerunning (resuming) the interrupted trials induced by the termination of an NNI experiment when resuming this experiment.

File: docs/source/release.rst (hunk 5 above)
Comment (Louis-J) [reply docs/source/release.rst:25]: [view tensors](https://pytorch.org/docs/stable/tensor_view.html) can be produced by many operators(30+, such as `expand`, `t`, or `detach`).

Comment (QuanluZhang) [reply docs/source/release.rst:25]: Got it. Is "...error with viewing (e.g., torch.view, torch.reshape) tensors..." better?

File: docs/source/release.rst [hunk 10]
```diff
@@ -5,6 +5,47 @@
 Change Log
 ==========
 
+Release 2.10 - 11/14/2022
+-------------------------
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (ultmaster) [inline docs/source/release.rst:12]: Title level is different in NAS and Compression. One of them must be wrong.

**Review by QuanluZhang — APPROVED** [review]

File: docs/source/release.rst [hunk 11]
```diff
@@ -5,6 +5,47 @@
...
+
+*  Added trail deduplication for evolutionary search.
```
Comment (ultmaster) [inline docs/source/release.rst:14]: trial

**Review by ultmaster — APPROVED** [review]
